performant-software/textlab
# Default user permission

**What This Does**
Closes #170 

**How to Test**
1 Go to: https://mel-textlab-pr-174.herokuapp.com/
2 Create a user
3 Log in as admin and activate but do not set user type
4 Verify that user appears as a "user" 
5 Log in as user and verify new account is not an administrator and has no access to admin functionality
